docs/scos/dev/upgradability-services/upgradability-reference.md

```diff
@@ -5,15 +5,81 @@ last_updated: Nov 25, 2021
 template: concept-topic-template
 ---
 
-## Method is overridden with a Factory, Dependency Provider, Repository, or Entity Manager extended
 
-### What is the nature of the upgradability error?
+## A core method is used on the project level
 
+Modules have public and private APIs. The public API includes the entities like facade, plugin stack, dependency list, etc. This check covers private API entities, including Business model, Factory, Dependency provider, Repository, and Entity manager.
 
-### Example of code that can cause the upgradability errors
+While public API updates always support backward compatibility, private API updates can break backward compatibility. So, backward compatibility in minor releases is not guaranteed in the private API. For example, if you use a core method on the project level, and it is updated or removed, the error may occur during an update.
```

Comment:
For example, if you use a core method on the project level, and it is updated or removed, it can cause unexpected issues during minor updates.